### PR TITLE
follow hierarchy and add migration

### PR DESCRIPTION
**Pull request by AshfaqUet**
In this commit I have add the migrations as well as I also follow a proper hierarchy of the flask project

### CODE REVIEW

File: app/__init__.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+from config import Config
+from flask_sqlalchemy import SQLAlchemy
+from flask_migrate import Migrate
+
+app = Flask(__name__)
+app.config.from_object(Config)
+db = SQLAlchemy(app)
+migrate = Migrate(app, db)
+
+from app import routes
```
Comment (kiranalii) [inline app/__init__.py:11]: Add extra line at the end of file

https://realpython.com/python-pep8/

**Review by kiranalii — COMMENTED** [review]
Please format your code according to pep8 standards. And add .pyc, .idea etc files to .ignore file. WIll review rest later

File: app/__init__.py (hunk 1 above)
Comment (AshfaqUet) [reply app/__init__.py:11]: ok I will do it

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
+FROM python:3.6-alpine
+
+RUN adduser -D filterlastname
+
+WORKDIR /home/filterlastname
+
+COPY requirements.txt requirements.txt
+#RUN python -m venv venv
+RUN pip install -r requirements.txt
+RUN pip install gunicorn
```
Comment (kiranalii) [inline Dockerfile:10]: Read about flask server and gunicorn

File: app/models.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+from app import db
+
+
+class File(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(200), nullable=False)
+    filename = db.Column(db.String(64), index=True)
```
Comment (kiranalii) [inline app/models.py:7]: index?

File: app/models.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+    def __repr__(self):
+        return '<name {}>'.format(self.name)
+
+    def toJson(self):
+        return {
+            "id":self.id,
+            "name":self.name,
+            "filename":self.filename
+        }
```
Comment (kiranalii) [inline app/models.py:17]: Add extra line and resolve pep8 errors

File: app/models.py [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+class File(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(200), nullable=False)
+    filename = db.Column(db.String(64), index=True)
+
+    def __repr__(self):
+        return '<name {}>'.format(self.name)
+
+    def toJson(self):
```
Comment (kiranalii) [inline app/models.py:12]: follow snake case for method naming

File: app/routes.py [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+from flask import request, render_template, jsonify
+from app import functions, models
+
+
+@app.route('/')
+def index():
+    return ("Hello its index route")
+
+
+@app.route('/uploads', methods=['POST'])     # route at which we uploads the file
```
Comment (kiranalii) [inline app/routes.py:12]: Difference between rest architecture and http

File: config.py [hunk 7]
```diff
@@ -0,0 +1,14 @@
+import os
+
+basedir = os.path.abspath(os.path.dirname(__file__))
+
+
+class Config(object):
+    SECRET_KEY = os.environ.get(
+        'SECRET_KEY') or 'you-will-never-guess'  # this is used for encryption purpose mostly app
```
Comment (kiranalii) [inline config.py:8]: read pep8 standard for multiple line

File: migrations/README [hunk 8]
```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```
Comment (kiranalii) [inline migrations/README:1]: Add extra line

**Review by kiranalii — APPROVED** [review]

**Pull request merged**